test/test_settings.py

```diff
@@ -333,7 +333,7 @@ def test_cannot_set_invalid_values_for_max_frame_size(self, val):
             assert e.value.error_code == h2.errors.ErrorCodes.PROTOCOL_ERROR
             assert s[h2.settings.SettingCodes.MAX_FRAME_SIZE] == 16384
 
-    @given(integers())
+    @given(strategies.integers())
```

Comment:
I did this since I was using more functions within the `strategies` submodule along with `integers`.  Probably  just a nervous tick I have 😁 .  I am happy to conform to whatever style you deem appropriate ie change this back and just import all the functions from the strategies submodule that I use.